src/standard/expr.mli

```diff
@@ -1182,6 +1182,9 @@ module Term : sig
     include Dolmen_intf.Term.Tptp_Tff_Arith_Common with type t := t
     (** Satisfy the common interface for TPTP's arithmetic on integers. *)
 
+    val pow : t -> t -> t
+    (** Exponentiation on integers. *)
+
```

Comment:
This (and probably a few lines after this), could be removed by adding an `include` for the `Ae` arith module types (similar to the includes for the smtlib and tptp arith module types just above)

Reply:
Done.